python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -123,6 +123,7 @@ def __init__(
                     initializer=init_global_processor,
                     mp_context=mp.get_context("fork"),
                     initargs=(server_args,),
+                    max_workers=server_args.image_processor_max_workers,
```

Comment:
The key point is obtaining the number of cores in the real operating environment, such as the Docker runtime environment. e.g.
```python3
import psutil
print(psutil.cpu_count(logical=False))
print(psutil.cpu_count(logical=True))
```

Reply:
this does not work in docker as well

Reply:
ok